docs/dev-standards/security.md

```diff
@@ -0,0 +1,33 @@
+# Security
+
+## Introduction
+
+Security is a core part of how we design, build and operate digital services. Applying consistent security principles helps teams make informed decisions, reduce risk, and build systems that are resilient by default.
+
+Security must be considered from the outset and throughout the software delivery lifecycle — not added as an afterthought.
```

Comment:
Markdown M-dash rather than literal character
```suggestion
Security must be considered from the outset and throughout the software delivery lifecycle --- not added as an afterthought.
```